samples/zigbee/network_coordinator/README.rst

```diff
@@ -16,17 +16,19 @@ It is a minimal implementation that supports only the network steering commissio
 Requirements
 ************
 
-* One of the following development kits:
+The sample supports the following development kits:
 
-  * |nRF52840DK|
-  * |nRF52833DK|
+.. include:: /includes/boardname_tables/sample_boardnames.txt
+   :start-after: set8_start
+   :end-before: set8_end
 
-* One or both of the following samples:
+You can use one or more of the development kits listed above and mix different development kits.
```

Comment:
I think here you use only one DK, not one or more?

Reply:
yes, there should be one network coordinator, not more

Reply:
Changed.